src/helm/proxy/models.py

```diff
@@ -833,6 +833,13 @@ def engine(self) -> str:
         name="stabilityai/stablelm-base-alpha-7b",
         tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG],
     ),
+
+    Model(
+        group="lightningai",
+        name="lightningai/lit-gpt",
+        tags=[TEXT_MODEL_TAG, INSTRUCTION_FOLLOWING_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, GPT2_TOKENIZER_TAG]
```

Comment:
Remove `GPT2_TOKENIZER_TAG` and `INSTRUCTION_FOLLOWING_MODEL_TAG` (I assume that not all models use the GPT-2 tokenizer and are instruction following)

Also replace `FULL_FUNCTIONALITY_TEXT_MODEL_TAG` with `LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG` for now. We need logprobs,  `num_completions` and `top_k_per_token` support for full functionality.